internal/app/machined/pkg/system/runner/containerd/containerd_test.go

```diff
@@ -424,3 +424,35 @@ func TestContainerdSuite(t *testing.T) {
 
 	suite.Run(t, new(ContainerdSuite))
 }
+
+func (suite *ContainerdSuite) TestContainerStdin() {
```

Comment:
nit: you might want it to put it before the `TestContainerdSuite()` func above to keep it aligned

Reply:
Ah yep :+1: